components/builder-web/app/profile/profile/profile.component.html

```diff
@@ -88,6 +88,40 @@ <h3>Personal Access Token</h3>
         </ng-container>
       </section>
       <hr>
+      <section>
```

Comment:
we only want this on SAAS.

Reply:
I went looking to see how this is done and ther are a couple spots where we display something only on saas. Look at the habitat.conf.jsfile and you will see two settings that are based on `cfg.hosted`.  You could add a thirs here for displaying the license. Or maybe it would be better just to have one setting that is `is_saas_builder` and consolidate all three usages under this.